languages/python/exercises/concept/string-methods/.docs/introduction.md

```diff
@@ -0,0 +1,16 @@
+## String Methods
+
+A `str` in Python is a sequence of Unicode code points which
+include letters, numbers, symbols, punctuation, etc. Strings
+implement all of the [common sequence operations](https:/docs.python.org/3/library/stdtypes.html#typesseq-common),
```

Comment:
As per [the docs](https://github.com/exercism/v3/blob/master/docs/concept-exercises.md#docsintroductionmd):

> Links should be used sparingly, if at all. While a link explaining a complex topic like recursion might be useful, for most concepts the links will provide more information than needed so explaining things concisely inline should be the aim.

I think this link can be removed here and instead be added as a hint.